x-pack/plugins/observability/public/pages/alerts/severity_badge.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBadge } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export interface SeverityBadgeProps {
+  severityLevel?: string;
+}
+
+const colorMap: { [key: string]: string } = {
+  critical: 'danger',
```

Comment:
I think we have a const for this, AlertSeverityLevel IIRC. No need to change that now but maybe a comment here is a good idea.